packages/serverless-workflow-service-catalog/package.json

```
@@ -21,7 +21,8 @@
     "@kie-tools-core/envelope-bus": "0.0.0",
     "@types/js-yaml": "^4.0.5",
     "js-yaml": "^4.1.0",
-    "openapi-types": "^7.0.1"
+    "openapi-types": "^7.0.1",
+    "lodash": "^4.17.21"
```

My local build failed because of missing dependency:
```suggestion
    "lodash": "^4.17.21",
    "@types/lodash": "^4.14.168"
```